modules/bucket/variables.tf

```diff
@@ -1,3 +1,8 @@
+variable "enable_access_logs" {
+  description = "Set to true to enable access logs, log bucket will have bucket shortname appended with 'access-logs'. Bucket 'ent-gcs-foo-dev-001' will have access logs in bucket 'ent-gcs-foo-access-logs-dev-001'"
```

Comment:
doc does not match the test case `"ent-gcs-%s-axlogs-dev-%s"`

Reply:
maybe a simple `logs` infix is better? Shorter than `access-logs` and easier to understand than `axlogs`?

Reply:
There's a mismatch between what's documented and what actually happens.
What do you think about the suggestion of using only a simple `logs` infix?